src/core/disk_params.cc

```diff
@@ -65,6 +65,9 @@ struct convert<seastar::internal::disk_params> {
         if (node["write_saturation_length"]) {
             mp.write_saturation_length = parse_memory_size(node["write_saturation_length"].as<std::string>());
         }
+        if (node["physical_block_size"]) {
```

Comment:
doc/io-properties-file.md needs to be updated

Reply:
thanks. updated.